app/operations/shopify/product_operations.rb

```diff
@@ -0,0 +1,27 @@
+module Shopify
+  class ProductOperations
+    class << self
+      def create(spree_product:)
+        service = Shopify::ProductExporter.new(spree_product: spree_product)
+        service.perform
+      end
+
+      def update(spree_product:)
```

Comment:
Then you would have to go check what the class takes as a parameter in order to know that you must pass a `spree_product`. Having it set as a keyword argument will explicitly raise an error saying that `spree_product` is a required argument.